packages/venia-ui/lib/components/ForgotPassword/forgotPassword.js

```diff
@@ -1,61 +1,64 @@
-import React, { Component, Fragment } from 'react';
-import PropTypes from 'prop-types';
+import React, { Fragment, useCallback } from 'react';
+import { bool, func, shape, string } from 'prop-types';
 
-import classify from '../../classify';
+import { mergeClasses } from '../../classify';
 import ForgotPasswordForm from './ForgotPasswordForm';
 import FormSubmissionSuccessful from './FormSubmissionSuccessful';
 import defaultClasses from './forgotPassword.css';
 
-class ForgotPassword extends Component {
-    static propTypes = {
-        classes: PropTypes.shape({
-            instructions: PropTypes.string
-        }),
-        completePasswordReset: PropTypes.func.isRequired,
-        email: PropTypes.string,
-        initialValues: PropTypes.shape({
-            email: PropTypes.string
-        }),
-        isInProgress: PropTypes.bool,
-        onClose: PropTypes.func.isRequired,
-        resetPassword: PropTypes.func.isRequired
-    };
-
-    handleFormSubmit = async ({ email }) => {
-        this.props.resetPassword({ email });
-    };
-
-    handleContinue = () => {
-        const { completePasswordReset, email, onClose } = this.props;
+const INSTRUCTIONS = 'Enter your email below to receive a password reset link.';
 
+const ForgotPassword = props => {
+    const {
+        completePasswordReset,
+        email,
+        initialValues,
+        isInProgress,
+        resetPassword,
+        onClose
```

Comment:
This is a `noop` now. Can we just remove it entirely?

Reply:
Yes, I'll either use it or remove it.

Reply:
This has been addressed; it's no longer a noop.